index.html

```diff
@@ -898,14 +898,166 @@ <h4 class="metrics-label">Dana Scully</h4>
                 </div>
             </li>
         </ul>
-        
+
         <style type="text/css">
             /* Funtivity CSS goes here */
+            .stat-table {
+                display: table;
+                color: #f46240;
+                width: 100%;
+            }
+            .stat-table > div {
+                display: table-cell;
+            }
+            .stat-table .info {
+                width: 60%;
+                font-weight: bold;
+            }
+            .stat-table .number {
+                font-size: 32px;
+                line-height: 34px;
+            }
+            .stat-table .label {
+                text-transform: uppercase;
+                font-size: 12px;
+            }
+            .stat-table .ss-icon {
+                font-size: 36px;
+                line-height: 36px;
+                vertical-align: middle;
+                text-align: center;
+                width: 40%;
+            }
+
+            /*
+              Base Style
+             */
+            .mini-stats-table .mini-stat {
+                display: inline-block
+            }
+            
+            .mini-stats-table .mini-stat .mini-stat-field, .mini-stats-table .mini-stat .mini-stat-value {
+                display: inline-block;
+            }
+
```

Comment:
Make it responsive.  
I used `!important` because i wanna make sure if any overrides are using borders, that in responsive view, it goes away.  Maybe?